app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/MainActivity.kt

```diff
@@ -134,6 +139,15 @@ class MainActivity : BaseActivity(),
         initUserEngagement()
         observeViewModel()
         onSessionAlarmNotificationTapped(intent)
+
+        if (Build.VERSION.SDK_INT >= 33) {
```

Comment:
Please use the associated constants, here `Build.VERSION_CODES.TIRAMISU` instead of `33`.

Reply:
:heavy_check_mark: Thx for applying this.